src/plugins/kibana/public/settings/sections/kibi_relations/controllers/relations.js

```diff
@@ -26,7 +26,7 @@ define(function (require) {
             scope.$apply(function () {
               ngModelCtrl.$setViewValue(elm.val());
             });
-          }, attr.ngDebounce || 1000);
+          }, attr.ngDebounce || 1);
```

Comment:
This was a hack I tried to just get the darn page to render without crashing, however now I think there's more to it than I first suspected, I haven't got too deep into it since there's been other things occupying my time since then. But basically this hack produced the desired result of rendering the page in a timely manner, however when clicking back n forth between the dashboard relations and index relations tabs, eventually the dashboard relations all have red X's next to the left and right dashboards effectively the source n target boards, n well this leads me to conclude that something else relies on the functions downstream and my hack breaks that. These X's result in being unable to save new Dashboard relations, so I'm guessing it's some kind of sanity check that I have still yet to look into, but basically this hack makes it so you can work in the dashboard relations tab if you do it quickly, but as soon as this sanity check or whatever finishes or fails to finish, they all go red and progress is halted, which forces a session clear and a refresh of the settings and the user to start back wherever they were before the X's appeared. So... I'll give this some time as soon as I have some, so I won't close this just yet, but maybe someone from Siren's side can lend insight into this before I begin reading all 1000 lines of this controller... 

also on the [line](https://github.com/sirensolutions/kibi/blob/master/src/plugins/kibana/public/settings/sections/kibi_relations/index.html#L186) you referenced, the var is `kibi-debounce="350"` so wouldn't `attr.kibiDebounce` be just as invalid?